sklearn/cluster/optics_.py

```diff
@@ -511,7 +538,8 @@ def cluster_optics_dbscan(reachability, core_distances, ordering, eps=0.5):
 
     eps : float, optional (default=0.5)
```

Comment:
I don't think `eps=0.5` is a reasonable default except on 2d toy examples.
I would suggest to rather not have a default, because users _need_ to choose the value appropriately for their data set.

Reply:
fair point, and true based on my few observations.